subsys/net/ip/net_context.c

```diff
@@ -1292,77 +1275,28 @@ NET_CONN_CB(tcp_synack_received)
 	 * If we receive SYN, we send SYN-ACK and go to SYN_RCVD state.
 	 */
 	if (NET_TCP_FLAGS(tcp_hdr) == (NET_TCP_SYN | NET_TCP_ACK)) {
-		struct sockaddr *laddr;
```

Comment:
Could we keep the addresses inside this block as there does not seem to be a reason why they would need to be outside?